### DISCUSSION TITLE
Where do I find the numbers of sent/received messages?

### DISCUSSION THREAD

**Discussion by stoicheio** [riebl/artery#343] [Q&A] [answered]
Hi,
I'm new to artery and it seems like a stupid question but i just can't find the numbers of sent/received messages. I'd like to compare the outcome of sent and received messages when modifying CAMs so any help is appreciated.
Thank you in advance!

Best regards

**Comment by riebl** [comment, marked as answer]
Hi @stoicheio,

The number of received and transmitted CAMs is not shown in the GUI but is collected as an OMNeT++ statistic and thus recorded in the result files.
In the *CaService.ned* file you can find two lines starting with `@statistic` for this purpose. The OMNeT++ manual describes this mechanism in great detail.

Best regards
Raphael

**Reply by stoicheio** [reply]
Hi Raphael,
thank you for your fast reply! This helps me a lot!

Best regards
